cuda/private/cuda_helper.bzl

```diff
@@ -243,7 +274,7 @@ def _create_common(ctx):
 
     merged_cc_info = cc_common.merge_cc_infos(cc_infos = [dep[CcInfo] for dep in all_cc_deps])
 
-    cc_toolchain = find_cpp_toolchain(ctx)
+    sysroot = _get_sysroot(ctx)
```

Comment:
I am quite sure that this is the wrong place and should be moved into `_impl` of `cuda_toolchain_config`. Currently this `get_memory_inefficient_command_line()` is called for every? `cuda_compile()` action.

Moving that would require plumbing through a new `sysroot` attribute and possibly the `sysroot` feature in `cuda_toolchain_config` has never been tested. So I hope I can leave that improvement for someone else.

Reply:
rebased on https://github.com/bazel-contrib/rules_cuda/pull/315